src/helpers/getSortedType.ts

```diff
@@ -0,0 +1,13 @@
+import { TodoStatus } from '../enums/TodoStatus';
+
+export const getTodoStatus = (type: string) => {
+  switch (type) {
+    case TodoStatus.Active:
+      return TodoStatus.Active;
+    case TodoStatus.Completed:
+      return TodoStatus.Completed;
+    case TodoStatus.All:
+    default:
```

Comment:
```suggestion
    case TodoStatus.Active:
      return TodoStatus.Active;
      
    case TodoStatus.Completed:
      return TodoStatus.Completed;
      
    default:
```
don't forget about empty lines:))
And i suppose you don't need case TodoSatus.All here